package lesson10;

/**
 * Created by Yevhen on 17.05.2017.
 */
//public class ExceptionExample4 {
//
////    public static void main(String[] args) {
////
////
////        public static void procA(){
////
////            try {
////                System.out.println("Inside procA");
////
////                throw new RuntimeException("demo");
////            } finally {
////                System.out.println("ProcA's finnally");
////            }
////
////        } разобраться или забить
////
////    }
////}